.circleci/config.yml

```diff
@@ -82,8 +82,8 @@ jobs:
     steps:
       - checkout
       - run:
-          name: Gometalinter tests
-          command: './scripts/gometalinter.sh'
+          name: linters tests
```

Comment:
`linters` might be a bit confusing since we have both backend and frontend linters.

Reply:
How about backendlinters?

Reply:
@bergquist Good point. I renamed it to `linters-backend`, similar to `test-frontend` and `test-backend`.